src/pocketmine/resources/pocketmine.yml

```diff
@@ -121,6 +121,50 @@ chunk-ticking:
  tick-radius: 3
  light-updates: false
  clear-tick-list: true
+ 
+block-ticking:
+ #Configure which blocks should tick and which not
+ #Useful on slow PC's and to disable ice melting, fire burning down things etc.
+ #Grass
```

Comment:
You can also do

```yaml
disable-block-ticking:
# grass
- 2
# blah
- 6
```

Nevertheless, users should read documentation if they want.